shell/common/gin_helper/locker.cc

```diff
@@ -17,6 +17,11 @@ void Locker::SetIsBrowserProcess(bool is_browser_process) {
   g_is_browser_process = is_browser_process;
 }
 
+void Locker::SetIsRendererProcess(bool is_renderer_process) {
+  g_is_renderer_process = is_renderer_process;
+}
+
 bool Locker::g_is_browser_process = false;
+bool Locker::g_is_renderer_process = false;
```

Comment:
`Locker` is the wrong place for this. `IsBrowserProcess` doesn't belong here and I don't want to propagate the error. Can we find a different way to do this?

Reply:
Hm, I could just move this to it's own helper? `ElectronProcess::IsBrowserProcess` and `ElectronProcess::IsRendererProcess`?

Reply:
Is there no existing API to get the current process type?